README.md

```diff
@@ -64,6 +64,35 @@ to the `require` section of your `composer.json`.
 Usage
 -----
 
+Configure socialNetworks in config file
+
+```php
+    'components' => [
```

Comment:
Can you remove this extra `Tab`, please?

Reply:
This is ordinary lvl of tab in main config. I think this is easily to include to config.

Reply:
@greeflas your opinion? We need to remove tab anyway?

Reply:
@bscheshirwork oh, I see what you mean. Let's keep it.